src/interface/chessBoardInterface.js

```diff
@@ -1,5 +1,57 @@
-export default class ChessBoardInterface {
-    constructor(parentNode, onFieldClick) {
+// export default class ChessBoardInterface {
+//     constructor(parentNode, onFieldClick) {
 
-    }
-}
\ No newline at end of file
+//     }
+// }
+
+var symbols = {
+    "rookWhite" : "\u2656",
+    "knightWhite" : "\u2658",
+    "bishopWhite" : "\u2657",
+    "queenWhite" : "\u2655",
+    "kingWhite" : "\u2654",
+    "pawnWhite" : "\u2659",
+
+    "rookBlack" : "\u265c",
+    "knightBlack" : "\u265e",
+    "bishopBlack" : "\u265d",
+    "queenBlack" : "\u265b",
+    "kingBlack" : "\u265a",
+    "pawnBlack" : "\u265f",
+};
+
+function ChessBoard() {
+
+    var board = document.querySelector('#board');
+
+    var cells = [];
+
+    this.createBoard = function () {
+         board.style.width = '400px';
+         board.style.height = '400px';
```

Comment:
Wymiary wpisane na sztywno w px i wrzucone z pozycji JS do inline style elementu? To bardzo zła praktyka - uzyskanie jakiejkolwiek responsywności staje się poważnym problemem. Lepiej ustawiać wymiary w pliku css.

Reply:
corrected